--- src/co2gui/__init__.py
+++ src/co2gui/__init__.py
@@ -268,7 +268,6 @@ def show_about(root, about_txt=about_txt, verbose=False):
     root.title("About %s" % APPNAME)
 
     textarea = tk.Text(root, wrap=tk.WORD,
-                       background='SystemButtonFace',
                        cursor='arrow')
     textarea.pack(fill=tk.BOTH, expand=1)
 
@@ -299,7 +298,7 @@ _sunken = dict(relief=tk.SUNKEN, padx=_pad, pady=_pad, borderwidth=_bw)
 
 def define_ttk_styles():
     style = ttk.Style()
-    style.configure('None.TButton', background='SystemButtonFace')
+    style.configure('None.TButton', )
     style.configure('True.TButton', foreground='green')
     style.configure('False.TButton', foreground='red')
     style.configure('TFrame', relief=tk.RAISED, padding=_pad)
@@ -308,7 +307,7 @@ def define_ttk_styles():
     style.configure('TA.TButton', foreground='orange')
     style.configure('Logo.TLabel')
     style.configure('Filepath.TButton', anchor='w')
-    style.configure('RO.Treeview', background='SystemButtonFace')  # NO, tags only
+    style.configure('RO.Treeview', )  # NO, tags only
     style.configure('OKDecision.TLabel', background='Green')
     style.configure('SAMPLEDecision.TLabel', background='Red')
 
@@ -1347,7 +1346,7 @@ class SimulatePanel(ttk.Frame):
         tree_frame.pack(side=tk.LEFT, fill=tk.BOTH, expand=1)
 
         tree = make_files_tree(tree_frame, selectmode='none', height=3)
-        tree.tag_configure('ro', background='SystemButtonFace')
+        tree.tag_configure('ro', )
         self.outputs_tree = tree
         add_tooltip(tree, 'out_files_tree')
 
@@ -1931,7 +1930,6 @@ and double-click on the result file to open it,
         [wdg:out-file]
         """)
         textarea = tk.Text(self, font='TkDefaultFont',
-                           background='SystemButtonFace',
                            foreground='olive',
                            cursor='arrow')
         textarea.pack(fill=tk.BOTH, expand=1)
@@ -2065,7 +2063,6 @@ class TemplatesPanel(ttk.Frame):
         [wdg:ipython-files]
         """)
         textarea = tk.Text(self, font='TkDefaultFont',
-                           background='SystemButtonFace',
                            foreground='olive',
                            cursor='arrow')
 
@@ -2153,7 +2150,6 @@ class DicePanel(ttk.Frame):
 
         """)
         textarea = tk.Text(self, font='TkDefaultFont',
-                           background='SystemButtonFace',
                            foreground='orange',
                            cursor='arrow',
                            wrap=tk.NONE)
@@ -2481,7 +2477,7 @@ class Co2guiCmd(cmdlets.Cmd):
 
     def _make_status(self, parent):
         status = tk.Text(parent, wrap=tk.NONE, height=1, relief=tk.FLAT,
-                         state=tk.DISABLED, background='SystemButtonFace')
+                         state=tk.DISABLED, )
         config_text_tags(status, LOGGING_TAGS)
 
         return status
